Formula/igv.rb

```diff
@@ -0,0 +1,23 @@
+class Igv < Formula
+  desc "Interactive Genomics Viewer"
+  homepage "https://www.broadinstitute.org/software/igv"
+  url "https://data.broadinstitute.org/igv/projects/downloads/2.4/IGV_2.4.1.zip"
+  sha256 "4be870db1f962aa580bedbfc9182d788dd2f5aada72d2fec27eae5b6080889db"
+  head "https://github.com/broadinstitute/IGV.git"
+
+  bottle :unneeded
+
+  depends_on :java
+
+  def install
+    inreplace "igv.sh", /^prefix=.*/, "prefix=#{prefix}"
+    prefix.install Dir["igv.sh", "*.jar"]
+    bin.install_symlink prefix/"igv.sh" => "igv"
+  end
+
+  test do
+    ENV.append "_JAVA_OPTIONS", "-Duser.home=#{testpath}/java_cache"
+    (testpath/"script").write "exit"
+    assert_match "Version", `#{bin}/igv -b script`
```

Comment:
It should be `shell_output("#{bin}/igv -b script", 1)` if that's the exit code on BrewTestBot. But we need a real test here.

Reply:
`igv` has a graphical interface, so not sure how we want to test that.

The `-b` flag does the follwing (http://software.broadinstitute.org/software/igv/startingIGV):
> -b <file>, --batch=<file> Immediately run the supplied batch script after start up.

I think that test is not so bad. We could also try to download a file from a server with it and run a batch script on it. Not sure it is worth it.

Reply:
yes a -b test would be good